tests/SpiffWorkflow/spiff/ServiceTaskTest.py

```diff
@@ -11,21 +12,35 @@
 from SpiffWorkflow.bpmn.exceptions import WorkflowTaskExecException
 from .BaseTestCase import BaseTestCase
 
-# TODO must be a better way to do this
 assertEqual = None
```

Comment:
I didn't understand why this was here, checked out the code, ran it, and it doesn't need to be here.  What is it for?!?!

Reply:
asking about the "AssertEqual = None"  - but commented on the wrong line.

Reply:
In the test case setUp, the global is set to `self.assertEqual` so that the mock connector can assert its name and params. Didn't realize you could omit this line and the `global assertEqual` on line 52 would create the variable.